indico/modules/rb_new/client/js/common/timeline/MonthlyTimelineContent.jsx

```diff
@@ -35,6 +35,12 @@ export default class MonthlyTimelineContent extends WeeklyTimelineContent {
         return [];
     }
 
+    getWeekendDays = () => (
```

Comment:
What about `get weekendDays() {...}`? Then you can use directly `this.weekendDays`.

Reply:
I'd use bracket style with return for this method. Using `=> ( ... )` over multiple lines without something that's obviously a return value (especially JSX) in the body isn't immediately clear without a closer look at the brackets. While looking at this PR I first thought there's a missing `return` statement